org.squidowl.halloy.json

```diff
@@ -0,0 +1,50 @@
+{
+    "app-id": "org.squidowl.halloy",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "halloy",
+    "finish-args": [
+        "--share=ipc",
+        "--filesystem=xdg-config",
+        "--filesystem=xdg-data",
```

Comment:
the linter doesn't like these
```
{
    "errors": [
        "finish-args-arbitrary-xdg-config-access",
        "finish-args-arbitrary-xdg-data-access"
    ]
}
```

Reply:
f72bb90 I've restricted access to our specific app folder